src/Entities/ApplicantEntity.php

```diff
@@ -2,10 +2,12 @@
 
 namespace Portal\Entities;
 
+use Portal\Interfaces\ApplicantEntityInterface;
 use Portal\Validators\EmailValidator;
+use Portal\Validators\PhoneNumberValidator;
 use Portal\Validators\StringValidator;
 
-class ApplicantEntity implements \JsonSerializable
+class ApplicantEntity implements \JsonSerializable, ApplicantEntityInterface
```

Comment:
why doesnt this extend the baseApplicantEntity? youv duplicated some of the getters and properties